server/JsDbg.Gdb/GdbDebugger.cs

```diff
@@ -20,6 +22,10 @@ class GdbDebugger : IDebugger {
         }
 
         public async Task Run() {

```

Comment:
The Run() function is called over and over right? If so, should we cache the result of LookupTypeSize?

Reply:
Or is it just called once and the loop below is doing all the work?

Reply:
Only once, see https://github.com/MicrosoftEdge/JsDbg/blob/jsdbg_gdb_support/server/JsDbg.Gdb/Program.cs#L34

Reply:
Ah yes. Of course...